app/src/main/java/org/simple/clinic/teleconsultlog/shareprescription/TeleconsultSharePrescriptionScreen.kt

```diff
@@ -191,7 +192,8 @@ class TeleconsultSharePrescriptionScreen constructor(
   }
 
   override fun renderPrescriptionDate(prescriptionDate: LocalDate) {
-    prescriptionDateTextView.text = dateFormatter.format(prescriptionDate)
+    // Since we need the date shown in prescription image be english, we are using US locale here
+    prescriptionDateTextView.text = dateFormatter.withLocale(Locale.US).format(prescriptionDate)
```

Comment:
We could also use `Locale.ENGLISH` here.

Reply:
Oh, TIL. 👍🏽